jobs/integr8ly/ocp4/ocp4-install.yaml

```diff
@@ -1,23 +1,31 @@
 ---
 
 - job:
-    name: pds-ocp4-install
+    name: ocp4-install
     project-type: pipeline
-    description: "Installs Integreatly on RHPDS OCP4 cluster using Integreatly Operator"
+    description: "Installs RHMI on specified OCP4 cluster using Integreatly Operator"
```

Comment:
[Minor] My understanding is that this is not for any random OCP4 cluster, but for either RHPDS cluster or a cluster we have credentials for stored in rhmi-jenkins repo. This might be somehow mentioned in the description.

Reply:
Thinking about it a bit more, it is either RHPDS or a cluster created by our (yours) pipelines. This should be clear so that people don't use this for any other clusters.